packages/oauth/package.json

```diff
@@ -33,7 +33,7 @@
     "build:clean": "shx rm -rf ./dist ./coverage ./.nyc_output",
     "lint": "eslint --ext .ts src",
     "test": "npm run lint && npm run test:mocha",
-    "test:mocha": "nyc mocha --config .mocharc.json src/*.spec.js",
+    "test:mocha": "nyc mocha --config .mocharc.json src/*.spec.js src/**/*.spec.js src/*.spec.ts src/**/*.spec.ts",
```

Comment:
As I've added more tests including the ones in TypeScript, this change is necessary for running all